models/MessageExport.php

```
@@ -6,49 +6,47 @@
+     * code      | default   | en    | de    | fr    | found
+     * --------------------------------------------------
+     * title     | Title     | Title | Titel | Titre | 1
+     * name      | Name      | Name  | Name  | Prénom| 0
      * ...
-     *
-     * @param $columns
-     * @param null $sessionKey
-     * @return mixed
      */
-    public function exportData($columns, $sessionKey = null)
+    public function exportData(array $columns, ?string $sessionKey = null): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$sessionKey'. (undefined)

(UnusedFormalParameter)